packages/dashboard/src/pages/Wallets/CreateWallet.tsx

```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { CreateWalletUI } from '../../../../maskbook/src/plugins/Wallet/UI/CreateWalletDialog'
```

Comment:
this import path?????

Reply:
Maybe bug of vscode

Reply:
Please do not import `maskbook` package in the dashboard package.

Reply:
> Please do not import maskbook package in the dashboard package.

Then we might need to find a way to organize sharable UI components from plugin.

Reply:
I know we have a plan to move plugins in an individual package, maybe we can start it sooner.

Reply:
I'm working on this